### DISCUSSION TITLE
Development plans?

### DISCUSSION THREAD

**Discussion by gamboz** [comments-ink/django-comments-ink#59] [Q&A]
I gently ask if there is any planning or time-schedule to bring the project to production/stable development status.

Also, is there some easy task that I could help with?

P.S. thank you for this project and all the tutorial and demo sites :)

**Comment by danirus** [comment]
It is funny that you mention the tutorials... they don't exist yet :-)

The demo site on `dci-project-stories` repository has some issues that would be great to fix. ie: when you click on the number of user reactions to the "Net Neutrality in Jeopardy" story (it is the number 77 at the bottom of the article) a javascript modal pops to display the list of users that clicked on the clapping icon. But it is broken. It needs to be fixed. It is something very specific of the dci-project-stories. The code is in the static directory.

I attach a screenshot of how the modal looks like.
![Screenshot 2024-04-08 at 15 02 54](https://github.com/comments-ink/django-comments-ink/assets/157103/4c90fed7-71b8-4ebd-a482-c624856841d9)

**Reply by gamboz** [reply]
I think it might be a problem of template-name miss-matching (and a `content_type` missing from the context).

Please check to see if these two might help:
- https://github.com/comments-ink/dci-project-stories/pull/7
- https://github.com/comments-ink/django-comments-ink/pull/60


Please forgive me if I insist, but do you have any plan or expectations about when django-comments-ink could be considered beta or production?  (of course, "no" is a perfectly acceptable answer :wink: )


P.S. for the tutorial, I had in mind that of django-comments-xtd :smile:

**Comment by danirus** [comment]
I miss:
* adding SASS to have a better way to deal with the CSS files, 
* adding RollupJS to bundle the javascript sources, and
* the docs

It is still a lot of work...
